cove_iati/rulesets/iati_standard_v2_ruleset/actual_dates_in_the_past.feature

```diff
@@ -1,17 +1,17 @@
 Feature: Actual dates must be in the past
 
   Scenario Outline: activity-date[@type='2'].@iso-date must be today or in the past
-    Given `activity-date[@type='2']` elements
-      Then `iso-date` attribute must be today or in the past
+    Given `.` elements
+      Then `activity-date[@type='2']/@iso-date` must be today or in the past
```

Comment:
This is nice if it does what I think it does in the rule step function (see my comment https://github.com/OpenDataServices/cove/pull/869/files#r145083188) as it keeps concise and quite readable.